other/recon/backend-language.bcheck

```diff
@@ -0,0 +1,21 @@
+#This BCheck uses regex patterns to detect which programming language the application is using
+#Verified: Yes
+metadata:
+    language: v1-beta
+    name: "Backend-language"
+    description: "Passive scan for the programming language used by the backend"
+    author: "Brumens"
+    tags: "passive", "live", "backend", "recon", "info", "lang", "language"
+
+define:
+    desc = "The programming language of the application has been discovered"
```

Comment:
This has only been added to make it more user-friendly in case another user using the script would like to add a custom correction. I instead comment it out